config/messages.py

```diff
@@ -38,7 +38,8 @@ class Messages(metaclass=Formatable):
     submod_plus_only = "Na tohle mají práva jen Submod+. <:KKomrade:484470873001164817>"
     mod_plus_only = "Na tohle mají práva jen Mod+. <:KKomrade:484470873001164817>"
     bot_admin_only = "Na tohle mají práva jen Admin. <:KKomrade:484470873001164817>"
-    vote_room_only = "Tohle funguje jen v {room}."
+    vote_room_only = "Tohle funguje jen v <#{room}>."
+    mod_room_only = "Tohle funguje jen v <#{room}>."
```

Comment:
Since these two share the same text, I would prefer to unify them and use the unified message only (see other suggestions).
```suggestion
    specific_room_only = "Tohle funguje jen v <#{room}>."
```